ipn/localapi/localapi.go

```diff
@@ -599,6 +599,8 @@ func (h *Handler) serveDebug(w http.ResponseWriter, r *http.Request) {
 			break
 		}
 		h.b.DebugNotify(n)
+	case "notify-repeat-last-netmap":
+		h.b.DebugNotifyLastNetMap()
```

Comment:
Repeat is slightly misleading as it suggests it repeats indefinitely but it's like Resend

Reply:
Went with just `notify-last-netmap`... it's shorter.

Reply:
Oh does this do the private key redaction correctly?

Reply:
Those are in prefs. So yes.